node/pkg/node/node.go

```diff
@@ -84,11 +84,21 @@ func GuardianOptionP2P(p2pKey libp2p_crypto.PrivKey, networkId string, bootstrap
 		}}
 }
 
+func GuardianOptionNoAccountant() *GuardianOption {
```

Comment:
Did you want to comment that this is a test only function?  Or doesn't it matter?

Reply:
it's not a test-only function; it's a convenience function. I added a docstring comment to this option and while I was at it I added comments to all the other options as well :)